src/components/container/DailyDataChart/DailyDataChart.tsx

```diff
@@ -146,10 +152,14 @@ export default function DailyDataChart(props: DailyDataChartProps) {
     const GraphToolTip = ({ active, payload, label }: any) => {
         if (active && payload && payload.length) {
             var date = payload[0].payload.date;
+            let formatter = props.valueFormatter;
```

Comment:
Minor, but can simplify to: 

`let formatter = props.valueFormatter ?? getDailyDataTypeDefinition(props.dailyDataType).formatter;`

Reply:
Fixed